examples/minimal.yml

```diff
@@ -0,0 +1,24 @@
+kernel:
+  image: "mobylinux/kernel:4.9.x"
+  cmdline: "console=ttyS0 console=tty0 page_poison=1"
+init:
+  - linuxkit/init:42fe8cb1508b3afed39eb89821906e3cc7a70551
+  - mobylinux/runc:b0fb122e10dbb7e4e45115177a61a3f8d68c19a9
+  - linuxkit/containerd:60e2486a74c665ba4df57e561729aec20758daed
+onboot:
+  - name: dhcpcd
+    image: "linuxkit/dhcpcd:48e249ebef6a521eed886b3bce032db69fbb4afa"
+    binds:
+     - /var:/var
+     - /tmp/etc:/etc
+    capabilities:
+     - CAP_NET_ADMIN
+     - CAP_NET_BIND_SERVICE
+     - CAP_NET_RAW
+    net: host
+    command: ["/sbin/dhcpcd", "--nobackground", "-f", "/dhcpcd.conf", "-1"]
+trust:
+  image:
+    - mobylinux/kernel
```

Comment:
if we update to linuxkit above, this will change too.  The `linuxkit/kernel:4.9.x` kernel is signed:
```
🐳 $ notary -s https://notary.docker.io list docker.io/linuxkit/kernel
NAME         DIGEST                                                              SIZE (BYTES)    ROLE
----         ------                                                              ------------    ----
<snip>
4.9.x        95d8f78534ccbab2ea86b66716333d58a758edca00e260a92e90d9ef9a5c7d39    738             targets/releases
```